pareto/examples/strategic_model_treatment_module_demo/strategic_model_treatment_module_demo.ipynb

```diff
@@ -0,0 +1,1662 @@
+    "\n",
+    "![Compoonent removal efficiency](data_input_treatment_removal_efficiency.png)\n",
+    "\n",
+    "NOTE: PARETO supports two different definitions for treatment component removal efficiency. See [this](https://pareto.readthedocs.io/en/latest/model_library/water_treatment/index.html#treatment-component-removal-efficiency) page for details."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Desalination technologies\n",
+    "In tab \"DesalinationTechnologies\", specify whether or not each technology is for desalination:\n",
+    "\n",
+    "![Desalination technologies](data_input_desalination_technologies.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Desalination sites\n",
+    "In tab \"DesalinationSites\", specify whether or not each site is avaialble for desalination technologies or non-desalination technologies:\n",
+    "\n",
+    "![Desalination sites](data_input_desalination_sites.png)\n",
+    "\n",
+    "If a treatment site is specified as a desalination site, then only desalination technologies may be built there. Similarly, if a treatment site is specified as a non-desalination site, then only non-desalination technologies may be built there."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Storage initial water quality\n",
+    "In tab \"StorageInitialWaterQuality\", specify the initial quality of the water in each storage site:\n",
+    "\n",
+    "![Storage initial quality](data_input_storage_initial_quality.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Pad storage initial water quality\n",
+    "In tab \"PadStorageInitialWaterQuality\", specify the initial quality of the water at the built-in storage at each completions pad:\n",
+    "\n",
+    "![Pad storage initial quality](data_input_pad_storage_initial_quality.png)\n",
+    "\n",
+    "NOTE: It is possible that in the future PARETO will no longer support built-in storage capacity at completions pads."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## <a id='solve'></a>Set up and solve Pyomo model\n",
+    "To begin, ensure that you are working in a development environment where PARETO is installed. For example, if you followed the PARETO [installation instructions](https://pareto.readthedocs.io/en/latest/getting_started/index.html) to create a conda environment called `pareto-env`, then make sure that environment is active by running `conda activate pareto-env`.\n",
+    "\n",
+    "NOTE: The code in this notebook uses the Gurobi commerical optimization solver by default. If you do not have Gurobi installed, it will use the free solver CBC instead. The code will be much slower running with CBC than with Gurobi.\n",
```

Comment:
somehow i feel we should remove `much` from the last line for solver classification

Reply:
Addressed.